### PR TITLE
{phys}[foss/2020a] XOOPIC v20210302

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)
Requires:
- [x] PR #15268

### CODE REVIEW

File: easybuild/easyconfigs/x/XOOPIC/XOOPIC-20210302-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+reconfigurable via an input file."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/rinku-mishra/xoopic/archive/%s' % local_commit]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['e81a2b7a527b905fd58afd080f45690963079b0a0493a55929d2a0ef1246d4e3']
+
+builddependencies = [
+    ('binutils', '2.34'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/XOOPIC/XOOPIC-20210302-foss-2020a.eb:23]: no need to explicitly specify `binutils` when using the `foss` toolchain
```suggestion
```

Comment (sassy-crick) [reply easybuild/easyconfigs/x/XOOPIC/XOOPIC-20210302-foss-2020a.eb:23]: Fixed, thanks for pointing that out.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/lexming/3f4fdca11f1049c1c5995f9078513f3b for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Pull request merged**